docs/entities.md

```diff
@@ -504,6 +512,12 @@ user.profile = { name: "John", nickname: "Malkovich" };
 Will be stored in a single database column as `{"name":"John","nickname":"Malkovich"}` value.
 When you'll load data from the database, you will have your object/array/primitive back via JSON.parse
 
+When assigning `"null"` (a string) to a `simple-json` type property, it will be treated as a string and stored in its
```

Comment:
I'm not sure this should be part of the documentation, should it?  If you pass a string in you get a string out - even though it's named any certain way